sharedbuffers/mapped_struct.py

```diff
@@ -421,12 +484,19 @@ def unpack_from(cls, buf, offs, idmap = None):
         else:
             pbuf = buf
         try:
-            if pbuf[offs] == 'm':
+            fs_type = pbuf[offs]
+            if fs_type == 'm' or fs_type == 'M':
                 # inline bitmap
-                if cython.compiled and offs+7 >= pybuf.len:
+                if fs_type == 'm':
+                    fs_size = 7
+                elif fs_type == 'M':
+                    fs_size = 15
+                else:
+                    raise ValueError("Unknown set type %r" % fs_type)
```

Comment:
isn't this check a bit unnecessary, since you've already tested for those types above ?

Reply:
Yes, but it's free and it may save your life if the code that makes it unnecessary changes